pkg/modules/giveaway_start.go

```diff
@@ -16,7 +16,7 @@ func (c *giveawayCmdStart) Trigger(parts []string, event pkg.MessageEvent) pkg.A
 	}
 
 	if c.m.emoteID == "" {
-		return twitchactions.Mention(event.User, "No emote ID set. Use '!25config emoteid 98374583' to configure this module")
```

Comment:
the "or" part is wrong, the emotename isn't the requirement we check here. it should be a separate `if c.m.emoteName == ""`

Reply:
Fucking thing deleted.
I've updated it, however I have some doubts. If emoteID is filled, but emoteName is not; would the bot not return "No emote name set. Use '!25config emotename NaM' to configure this module"? Would this not require an `else if` statement of some sort?

Reply:
no, since we use return to break out of the function entirely